datasets/superb/superb.py

```diff
@@ -183,6 +183,64 @@ class Superb(datasets.GeneratorBasedBuilder):
             url="https://www.tensorflow.org/datasets/catalog/speech_commands",
             data_url="http://download.tensorflow.org/data/{filename}",
         ),
+        SuperbConfig(
+            name="ic",
+            description=textwrap.dedent(
+                """\
+            Intent Classification (IC) classifies utterances into predefined classes to determine the intent of
+            speakers. SUPERB uses the Fluent Speech Commands dataset, where each utterance is tagged with three intent
+            labels: action, object, and location. The evaluation metric is accuracy (ACC)."""
+            ),
+            features=datasets.Features(
+                {
+                    "file": datasets.Value("string"),
+                    "speaker_id": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                    "action": datasets.ClassLabel(
+                        names=["activate", "bring", "change language", "deactivate", "decrease", "increase"]
+                    ),
+                    "object": datasets.ClassLabel(
+                        names=[
+                            "Chinese",
```

Comment:
Should the language labels be lowercase for consistency with the other classes (or vice versa)?

Reply:
I'm leaning towards leaving these as-is, since these are the original class names and `s3prl` uses them in this format too.

Reply:
Perfect, I agree it is good to stay close to the raw data where possible :)